#
# Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved.
#
# This patch fixes: Bug 16976396 - [XKB] missing "olpc" variant from "af" layout
#
--- a/rules/0001-lists.part	2022-10-20 06:19:06.759167350 -0700
+++ b/rules/0001-lists.part	2022-10-20 06:19:23.570194791 -0700
@@ -18,7 +18,7 @@
 ! $tmgeometries = tm2020 tm2030PS2 tm2030USB tm2030USB-102 tm2030USB-106
 
 // Country files that provide specializations for the OLPC:
-! $olpclayouts = af am ara br ca es et fr in it kh kz mn ng np pk ru th tr us
+! $olpclayouts = am ara br ca es et fr in it kh kz mn ng np pk ru th tr us
 
 ! $applealu = applealu_ansi applealu_iso applealu_jis
 
